db/version_set.h

```diff
@@ -374,9 +375,10 @@ class VersionSet {
   {
       bool m_Submitted;     //!< level submitted to hot thread pool
       bool m_Running;       //!< thread actually running compaction
+      uint64_t m_LastCompaction; //!<NowMicros() when last compaction completed
 
       CompactionStatus_s()
-      : m_Submitted(false), m_Running(false)
+      : m_Submitted(false), m_Running(false), m_LastCompaction(Env::Default()->NowMicros())
```

Comment:
Hmm...I think the "initialize to 0" is in the wrong place. Should be in the ctor for CompactionStatus_s. With this change, SetCompactionDone() will always set m_LastCompaction to 0, which means the calculation in VerstionSet::Finalize() will always set elapsed_micros to 0.